changelog/v0.15.0/knative-fixes.yaml

```
@@ -0,0 +1,16 @@
+changelog:
+  - type: FIX
+    description: fix missing hosts on clusteringress proxy translation
+    issueLink: https://github.com/solo-io/gloo/issues/790
+  - type: FIX
+    description: provide rbac for updating clusteringress/status
+    issueLink: https://github.com/solo-io/gloo/issues/789
+  - type: FIX
+    description: enable http2 for ports with prefix http2
+    issueLink: https://github.com/solo-io/gloo/issues/788
+  - type: BREAKING_CHANGE
+    description: expose use_http2 field on upstream spec. the use_http2 field has been removed from the Static UpstreamSpec
```

```suggestion
    description: Expose use_http2 field on upstream spec. The use_http2 field has been removed from the Static UpstreamSpec
```

I know that this is really nit-picking, but since these get exposed on the release page, I think it looks better if we make these descriptions grammatically correct

---

(Applies to all the other entries here as well)